config/config_python.py

```diff
@@ -1,37 +1,20 @@
 import pip
 import pkg_resources
-# The collection of required packages.
-# Packages with value True will be updated if it's already installed.
-# Packages with value False will be skipped if it's already installed.
-packages = {'pyyaml': True}
+# The collection of required packages
+packages = ['pyyaml']
 
-def main(packages):
+def main(packages, upgrade = False):
     # Install gslab_tools if it's not installed yet or it has the wrong version
-    if not check_gslab_tools('4.1.0'):
-        pip.main(['install', 'git+http://git@github.com/gslab-econ/gslab_python.git@4.1.0'])
+    pip.main(['install', 'git+http://git@github.com/gslab-econ/gslab_python.git@4.1.0'])
```

Comment:
I think we want the `--upgrade` flag here too. Also, we normally release with the tag `v4.1.0`; so change that too.